docs/user/security/session-management.asciidoc

```diff
@@ -0,0 +1,60 @@
+[role="xpack"]
+[[xpack-security-session-management]]
+=== Session management
+
+When you log in to {kib} it creates a session that is used to authenticate subsequent requests to {kib}. A session consists of two components: an encrypted cookie that is stored in your browser, and an encrypted document in a dedicated {es} hidden index. By default, the name of that index is `.kibana_security_session_1` where the prefix is derived from {kib}'s primary `.kibana` index.
+
+When your session expires, or you log out of {kib} explicitly it will invalidate your cookie and remove session information from the index. {kib} also periodically invalidates and removes any expired sessions that weren't invalidated explicitly.
+
+[[session-idle-timeout]]
+==== Session idle timeout
+
+You can optionally expire sessions after a period of inactivity. By default, a session stays
+active until the browser is closed. To define a sliding session expiration, set
+the `xpack.security.session.idleTimeout` property in the `kibana.yml`
+configuration file. The idle timeout is formatted as a duration of
+`<count>[ms|s|m|h|d|w|M|Y]` (e.g. '70ms', '5s', '3d', '1Y'). For example, set
```

Comment:
I'd suggest changing this to match the new description in my top-level comment, if you agree with it.